apps/aecore/lib/aecore/channel/channel_state_on_chain.ex

```diff
@@ -246,9 +271,9 @@ defmodule Aecore.Channel.ChannelStateOnChain do
              inspect(Poi.calculate_root_hash(poi))
            }"}
 
-        channel.sequence >= offchain_tx.sequence ->
+        channel.sequence > offchain_tx.sequence ->
```

Comment:
Why was the check changed? I don't think equal sequence is a valid slash.

Reply:
I checked in epoch. As stated in line 107 of `aesc_utils.erl` I believe we should have `>=` here.

Reply:
I changed it in order to allow solo closing a channel which was snapshoted(E.g. Peers A and B open a channel, A transfers funds to B, peer B snapshots the state and then disappears). As we have the force progress transaction we can force an update and then solo close  with the forced state, but this will require 3 txs instead of 2 txs.